core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -260,6 +325,21 @@ object StorageTool extends Logging {
         s"${MetadataVersion.IBP_3_0_IV0}; the default is ${MetadataVersion.LATEST_PRODUCTION}")
   }
 
+  private def addFeatureDependenciesParser(subparsers: Subparsers): Unit = {
+    val featureDependenciesParser = subparsers.addParser("feature-dependencies")
+      .help("Look up dependencies for a given feature version. " +
+        "If the feature is not known or the version not yet defined, an error is thrown. " +
+        "Multiple features can be specified."
+      )
+
+    featureDependenciesParser.addArgument("--feature", "-f")
+      .required(true)
+      .help("The features and their versions to look up dependencies for, in feature=level format." +
```

Comment:
should we be consistent with calling it version or level here?

Reply:
yes makes sense